webpack.config.js

```diff
@@ -1,6 +1,8 @@
 // #############
 // SETUP
 
+const BASE_PROJECT_NAME = 'skeleton'; // @TODO Replace this with your Django project name.
```

Comment:
No need for the comment. Any occurence of the work skeleton will be replaced with the app name, eg. skeleton becomes power_of_play and SKELETON becomes POWER_OF_PLAY.